app/views/smart_answers/inputs/_country_select_question.html.erb

```diff
@@ -1,7 +1,9 @@
+<span class="govuk-caption-xl"><%= @presenter.title %></span>
 <%= render "govuk_publishing_components/components/select", {
   id: "response",
   label: question.title,
-  heading_size: 'm',
+  heading_size: 'xl',
+  is_page_heading: true,
   error_message: question.error,
   options: question.select_options
 } %>
```

Comment:
Not blocking this PR - but should we add in the `heading_caption` attribute into the select component in the publishing components?